app/assets/javascripts/discourse/app/controllers/user-activity.js

```diff
@@ -1,18 +1,13 @@
 import Controller, { inject as controller } from "@ember/controller";
 import I18n from "I18n";
-import { alias } from "@ember/object/computed";
-import { exportUserArchive } from "discourse/lib/export-csv";
 import { inject as service } from "@ember/service";
 import discourseComputed, { observes } from "discourse-common/utils/decorators";
 
 export default Controller.extend({
   dialog: service(),
   application: controller(),
-  user: controller(),
   userActionType: null,
 
-  canDownloadPosts: alias("user.viewingSelf"),
```

Comment:
It seems like this is still being used so we should avoid removing it 

https://github.com/discourse/discourse/blob/e66fa8c48b6b8db211f95b1479debd7d9853d509/app/assets/javascripts/discourse/app/templates/user/activity.hbs#L81

Reply:
Also, I think we still need this condition when displaying the new section as well.